dags/multipod/maxtext_end_to_end.py

```diff
@@ -35,18 +35,23 @@
     catchup=False,
 ) as dag:
   test_name_prefix = "maxtext"
-  test_models = ["llama2", "mistral", "gamma"]
+  test_models = {
+      "llama2": ["test_llama2", "llama_finetuning_test"],
```

Comment:
Thanks Anisha! Do you think it's better to just add this [script](https://github.com/google/maxtext/blob/main/end_to_end/llama_finetuning_test.sh) into [test_llama2](https://github.com/google/maxtext/blob/main/end_to_end/test_llama2.sh)? So no change is needed in this file.

cc @jonb377 as code owner

Reply:
I'd say it's better to keep these tests independent, so the `end_to_end` tests don't require any configuration.

Reply:
Could you please explain the thought a bit more @jonb377 ?

Reply:
Oh I just mean we should keep all the configuration in `maxtext` and run the test using `bash end_to_end/<test_script>` here in XLML. If we consolidate the two scripts, we'd need to pass in some configuration to the test script at runtime.

I like what you have here @A9isha

Reply:
I am happy to approve this PR! I was thinking if it will be better if we consolidate two scripts, as `test_llama2` already has test for training, decode, and others. Wondering what's the purpose to separate fine-tuning to a different script? Or will it be better if we organize it in a way, i.e. separate training or decode from the test_llama2 as well?

I think it's @jonb377's decision.

Reply:
Let us discuss this offline. Let me start a thread